src/commands/signzone.rs

```diff
@@ -1418,41 +1419,73 @@ impl SignZone {
         Ok(records)
     }
 
-    fn find_apex(
-        records: &SortedRecords<StoredName, StoredRecordData, MultiThreadedSorter>,
-    ) -> Result<(StoredName, Class, Ttl, Serial), Error> {
-        let soa = match records.find_soa() {
-            Some(soa) => soa,
-            None => {
-                return Err(Error::from("Invalid zone file: Cannot find SOA record"));
+    // TODO: Add tests.
```

Comment:
Should this TODO stay?

Reply:
Let me see what code coverage says.

Reply:
<img width="824" alt="image" src="https://github.com/user-attachments/assets/db9652f0-fe02-448a-8608-76c1bb6001a7" />

Quite a bit of code not covered by existing tests.

Reply:
I'll close these testing gaps.